lambda.js

```diff
@@ -499,12 +508,45 @@ stork.status = (event, context, callback) => {
         console.log(`headers ${JSON.stringify(sanitized.headers)}`);
         console.log(`body ${sanitized.body}`);
 
+        console.log(uri);
+        console.log(config);
         return got.post(uri, config);
       })
       .then(() => callback())
       .catch((err) => {
-        console.log(err);
-        callback(err);
+        const shaToken = jwt.sign(
```

Comment:
👋 @vsmart 
I got this from https://github.com/mapbox/stork/blob/shaerror/lambda.js#L32-L49
This goes into `got.get` as the second argument.

Reply:
Ah. Nice catch! 👍  I wonder if we could outsource this logic into a generateShaToken function that we can reuse.

Reply:
Why do you need to create another github token? Why can't you continue to use the one that was generated in the initial round of requests?